tests/unit/builders/indexable-post-builder-test.php

```diff
@@ -163,9 +176,13 @@ protected function open_graph_image_set_by_user( $image_meta ) {
 	 * @covers ::__construct
 	 */
 	public function test_constructor() {
+		$this->assertInstanceOf(
```

Comment:
I prefer the use of the method `assertAttributeInstanceOf`

Reply:
`assertAttributeInstanceOf` is deprecated in PHPUnit 8.x and will be removed completely in PHPUnit 9. Juliette already replaced them from the existing unit tests [in this commit](https://github.com/Yoast/wordpress-seo/commit/ef7c905ff80504625c4467cf99c101190da4d628), so I opted to do the same here 🙂.